### PR TITLE
Added Sphido, updated File Sharing section with new sub category

### PR DESCRIPTION
**Pull request by Kovah**
* Reordered the file sharing section to list all sub categories alphabetically and added the new `Archiving` subcategory with a first application
* added Sphido to the CMS section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ Table of Contents
     - [E-commerce](#e-commerce)
   - [Federated Identity/Authentication](#federated-identityauthentication)
   - [Feed Readers](#feed-readers)
-  - [File Sharing and Synchronization](#file-sharing-and-synchronization)
-      - [Web based file managers](#web-based-file-managers)
+  - [File Sharing, Synchronization and Archiving](#file-sharing-and-synchronization)
```
Comment (nodiscc) [inline README.md]: Please revert this change and  create a top-level section named `Document Archiving` with Paperless as it's single entry. Please also mind adding this section to the table of contents :) I don't think it should be a sub-category as archiving and synchronization software are very different in nature.

**Comment by nodiscc** [comment]
Thanks a lot for the reordering and for adding Sphido. About Paperless, can you address the commit comment above? Thanks again

**Comment by Kovah** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -386,16 +393,18 @@ _Recipe management_
 
 
 
-## File Sharing and Synchronization
+## File Sharing, Synchronization and Archiving
```
Comment (nodiscc) [inline README.md]: Please restore the original section name (in TOC as well)

**Comment by nodiscc** [comment]
you forgot this ^

**Comment by Kovah** [comment]
:+1:

**Comment by nodiscc** [comment]
Nope :) You also need to restore the original section header `File Sharing and Synchronization`

**Comment by Kovah** [comment]
Last update. Should be final version now. :smile:

**Comment by nodiscc** [comment]
Thanks
